_pages/getting-started/equipment.md

```diff
@@ -13,6 +13,12 @@ This page is generally about Macs. If you have a Windows device, see [InSite](ht
 
 Badges are shipped to your local GSA office. You will receive an email once your badge has been shipped. At that point, you can make an appointment to pick it up. Your badge may be available during your orientation -- send an email to [hspd12.security@gsa.gov](mailto:hspd12.security@gsa.gov) to save yourself a trip. You may also ask your orientation leader to show you where to pick it up.
 
+## Contractors
+
+Depending on the work they will perform, contractors may receive GSA provided computers or equipment. If the contractor needs access to applications that require VPN, they are required to use a GSA provided computer. If they can complete their work using VDI, they do not need a GSA provided computer and may use their company issued computer.
```

Comment:
We should have this info in one place (the Contractors page or the Equipment one) and link from the other. No string feelings on which.

Reply:
Agreed -- going to take it to Contractors.